flink-cdc-connect/flink-cdc-pipeline-connectors/flink-cdc-pipeline-connector-kafka/src/main/java/org/apache/flink/cdc/connectors/kafka/json/debezium/DebeziumJsonSerializationSchema.java

```diff
@@ -185,14 +190,21 @@ public byte[] serialize(Event event) {
         }
     }
 
+    /**
+     * Refer to <a
+     * href="https://debezium.io/documentation/reference/1.9/connectors/mysql.html">...</a>.
```

Comment:
Hyperlink text [`...`](https://debezium.io/documentation/reference/1.9/connectors/mysql.html) could be replaced with some meaningful words, like:

```suggestion
     * Refer to <a
     * href="https://debezium.io/documentation/reference/1.9/connectors/mysql.html">Debezium docs</a> for more details.
```

Reply:
Addressed it.